### PR TITLE
Create Forums Stub

### PR DESCRIPTION
**Pull request by Gamingzone123**
create stub for forums article ()

Have hit a block to writing the article, i cant think of how to adhere to the change req even with suggestions (on hold until inspiration hits...)

anyone is free to contiue where i left off

contributes to issue #3150 
project: https://github.com/ppy/osu-wiki/projects/3

TODO: address change requests

### CODE REVIEW

File: wiki/Forum/en.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+stub: true
+tags:
+  - forum
```
Comment (TicClick) [inline wiki/Forum/en.md:4]: the tags don't need to duplicate the article's title

File: wiki/Forum/en.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+#Forum
+
+The forums are a place where the [osu!](/wiki/Disambiguation/osu!) community can discuss a variety of topics.
```
Comment (TicClick) [inline wiki/Forum/en.md:9]: that's still too short, mind adding a few more sentences?

File: wiki/Forum/en.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+tags:
+  - forum
+---
+
+#Forum
+
+The forums are a place where the [osu!](/wiki/Disambiguation/osu!) community can discuss a variety of topics. 
+
+The [forum index](https://osu.ppy.sh/community/forums) is generalised under a hierarchy:
+Forum -> Category (e.g; osu!) -> Section (e.g; development) -> Subforums (e.g; completed projects) -> Individual threads
```
Comment (TicClick) [inline wiki/Forum/en.md:12]: what I wrote in the review was just a comment with my idea, it shouldn't be put as-is into the article

File: wiki/Forum/en.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+One of ther main functions of the forums is to assist with [beatmap moderation](/wiki/Modding) where users get feedback and help to improve their beatmaps.
```
Comment (TicClick) [inline wiki/Forum/en.md:14]: this is not true anymore for a few years at least, all modding takes place in beatmap discussions

**Review by TicClick — CHANGES_REQUESTED** [review]
most if not all points from my early review still stand

**Comment by Gamingzone123** [comment]
Have hit a block with writing the article, on hold for now

**Comment by Walavouchey** [comment]
superseded by https://github.com/ppy/osu-wiki/pull/6454

**Pull request closed without merging**